hack/kind/vault/vault-init-job.bash

```diff
@@ -0,0 +1,40 @@
+#!/bin/sh
```

Comment:
Here is the shellcheck output for this script:

```
In /tmp/test.sh line 3:
set -eu -o pipefail
           ^------^ SC2039: In POSIX sh, set option pipefail is undefined.


In /tmp/test.sh line 30:
  vault kv put $path $key=$value
               ^---^ SC2086: Double quote to prevent globbing and word splitting.
                     ^--^ SC2086: Double quote to prevent globbing and word splitting.
                          ^----^ SC2086: Double quote to prevent globbing and word splitting.

Did you mean:
  vault kv put "$path" "$key"="$value"

For more information:
  https://www.shellcheck.net/wiki/SC2039 -- In POSIX sh, set option pipefail ...
  https://www.shellcheck.net/wiki/SC2086 -- Double quote to prevent globbing ...
```

Reply:
yes will do, sorry should have done by myself!

Reply:
they are now fixed.